docs/man_pages/publishing/appmanager-groups.md

```diff
@@ -0,0 +1,46 @@
+appmanager groups
+==========
+
+Usage | Synopsis
+------|-------
+General | `$ appbuilder appmanager groups`
+
+Lists all available user groups from Telerik AppManager. <% if(isHtml) { %>If you have not managed distribution groups from [Telerik AppManager](https://platform.telerik.com/appmanager), a single 'Default Group' should be provided.
+
+For more information about AppManager distribution groups, see [Adding AppManager Distribution Groups](http://docs.telerik.com/platform/appmanager/appmanager-portal/managing-groups/adding-distribution-group).<% } %>
+
+<% if(isConsole) { %>
+<% if(isMobileWebsite) { %>WARNING: This command is not applicable to mobile website projects. To view the complete help for this command, run `$ appbuilder help appmanager groups`<% } %>
```

Comment:
I'll be thankful, if someone more familiar with the help commands and documentation,  could take a look over the .md files. Because I do not know your best practices when writing help.